### ISSUE TITLE
I cant run applications outside of oculus home

### ISSUE THREAD

**Issue by Crusage** [LibreVR/Revive#582]
I dont know if this is possible, but please make it possible. Opening games without them being inside oculus home.

**Comment by Armada651** [comment]
I don't understand your question? Revive doesn't support Oculus Home, so how are you even opening games inside Oculus Home?

Just to clarify, this is Oculus Home:

![9139896_orig](https://cloud.githubusercontent.com/assets/2028870/25314839/732fe5a2-284b-11e7-89b8-b0683222c151.png)

**Comment by Crusage** [comment]
I meant, is it possible to open exe files that are meant to be played with the oculus touch/ oculus rift. But playing it with revive, HTC vive?

**Comment by Armada651** [comment]
Yes, that's explained in the manual: https://github.com/LibreVR/Revive/wiki#standalone-games

**Comment by SimplexPL** [comment]
I was always curious whether it would be possible to see Oculus Home 3D environment in the Vive - is it blocked by oculus, or not supported because it's not necessary?

**Comment by Armada651** [comment]
It's not supported because Oculus Home uses a bunch of private APIs that Revive doesn't implement. And there's no priority to reverse engineer and implement them because it's not necessary.
